packages/ui/src/ui/pages/tablet_cell_bundles/TabletCellBundles.tsx

```diff
@@ -226,6 +208,53 @@ export default function TabletCellBundles() {
     );
 }
 
+function ActiveBundleDetails({
+    activeBundle,
+    allowEdit,
+    cluster,
+}: {
+    activeBundle?: string;
+    allowEdit?: boolean;
+    cluster: string;
+}) {
+    const dispatch = useDispatch();
+    const allowAccounting = useSelector(getClusterUiConfigEnablePerBundleTabletAccounting);
+
+    const showEditor = React.useCallback(() => {
+        if (activeBundle) {
+            dispatch(showTabletCellBundleEditor(activeBundle));
+        }
+    }, [activeBundle, dispatch]);
+
+    return (
```

Comment:
Here you can simplify a little:
`if(! activeBundle) return <BundleMetaTable />`
This will slightly reduce nesting and simplify readability.